### PR TITLE
SVGO dependency

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | - |
| Decisions | Added svgo dependency |
| Animated GIF | - |

### CODE REVIEW

**Comment by gcandal** [comment]
Cool 👍 . That you confirm that the icons appear correctly after this?

File: package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
         "lint-all": "npm run lint",
         "lint-fix": "eslint \"./**/*.js\" --fix",
         "prettier": "prettier \"./**/*.{js,json}\" --write",
-        "pretty": "npm run prettier && npm run lint-fix",
+        "svgo": "svgo -r . --disable='removeDimensions,removeViewBox,minifyStyles'",
+        "pretty": "npm run prettier && npm run lint-fix && npm run svgo",
```
Comment (gcandal) [inline package.json]: Sort alphabetically

**Comment by gcandal** [comment]
> Cool 👍 . That you confirm that the icons appear correctly after this?

@NFSS10 ⬆️ ?

**Comment by NFSS10** [comment]
> > Cool 👍 . That you confirm that the icons appear correctly after this?
> 
> @NFSS10 ⬆️ ?

Yes, atleast in storybook it seems ok.

File: react/assets/icons/id.svg [hunk 2]
```diff
@@ -1,15 +1 @@
...
-        </g>
-        <path fill="#FFF" fill-rule="nonzero" d="M4 9a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0M2.6 21.5h-.2c-.774 0-1.4-.7-1.4-1.564v-5.872c0-.864.626-1.564 1.4-1.564h.2c.774 0 1.4.7 1.4 1.564v5.872C4 20.8 3.374 21.5 2.6 21.5"/>
-    </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="18" height="27" viewBox="0 0 18 27"><defs><path id="a" d="M0 .061h17.805v25.707H0z"/></defs><g fill="none" fill-rule="evenodd"><path d="M-8-2h32v32H-8z"/><g transform="translate(0 .5)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path fill="#FFF" fill-rule="nonzero" d="M6.924.06H.873C.391.06 0 .524 0 1.093v.384c0 .557.378 1.02.849 1.021h.009c4.958 0 8.66 3.084 9.253 11.189.037.503-.28.947-.706.997l-1.39.163c-.11.013-.193.126-.19.257.098 4.942-2.485 8.06-6.967 8.06H.793c-.44.005-.793.435-.793.955v.697c0 .527.361.953.807.953h6.118c6.009 0 10.88-5.754 10.88-12.853S12.936.06 6.926.06" mask="url(#b)"/></g><path fill="#FFF" fill-rule="nonzero" d="M4 9a1.5 1.5 0 11-3 0 1.5 1.5 0 013 0M2.6 21.5h-.2c-.774 0-1.4-.7-1.4-1.564v-5.872c0-.864.626-1.564 1.4-1.564h.2c.774 0 1.4.7 1.4 1.564v5.872C4 20.8 3.374 21.5 2.6 21.5"/></g></svg>
```
Comment (joamag) [inline react/assets/icons/id.svg]: this icon must be moved (it's external)

**Comment by joamag** [comment]
@NFSS10 I'm going to run the renaming myself. Merging this one

**Pull request merged**